Read habitatdata

ToonHub opened the pull request:
Add function to read habitatmap

florisvdh commented on R/read_habitatdata.R:
```diff
@@ -150,3 +152,215 @@ read_habitatmap_stdized <-
+#' Return the data source \code{habitatmap}
+#'
+#' \code{read_habitatmap} returns all polygons of the \code{habitatmap} (De Saeger et al., 2018) that (partially)
+#' contain habitat or a Regional Important Biotope (RIB).
```
last line to be skipped

ToonHub commented:
I changed variable names
I added an extra parameter select_hab. If TRUE only hab/rib is selected. Default value is FALSE.

ToonHub commented:
I converted most string variables to characters

st_read(..., stringsAsFactors = FALSE) didn't seem to work,.. so I converted the factors to characters after reading the shapefile

hansvancalster commented:
You can use `read_sf` instead if `st_read` to have stringsAsFactors = FALSE and return a tibble by default.

florisvdh commented:
Great additions @ToonHub, thanks!
I suggest to merge!
